conda-store/pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 dependencies = [
    "rich",
    "click",
-   "yarl",
+   "yarl<1.9.0",
```

Comment:
I'd prefer not to have to pin to a lower version. Is the `str(...) + '/'` the only viable alternative?

Reply:
I will say I've tried several approaches and it's been pretty disappointing :frowning_face: . I'd prefer if we used the str(...) + '/' method just to avoid having to pin yarl. Could you create a function to wrap these like `def _ensure_slash(...)` or something along those lines.

Reply:
Ok, updated @costrouc - possibly not the most elegant solution but let me know if I can change anything to suit this better.